code/FulpstationCode/blueshift_gauntlets/bsg.dm

```diff
@@ -0,0 +1,105 @@
+var/isPhased = FALSE
+var/isImplementSpawned = FALSE
+var/isDrapeSpawned = FALSE
+var/isCauterySpawned = FALSE
+var/bsgtrait = TRAIT_BLUESPACE_SHIFTING
+
+//Action Buttons
+var/datum/action/set_phase/setphase = new/datum/action/set_phase()
+var/datum/action/summon_implements/summon_implements = new/datum/action/summon_implements()
+
+/datum/action/set_phase //Create the button for toggling the gloves phase.
+	name = "Change Phase - Enable or disable the gloves phasing device"
+	icon_icon = 'icons/mob/actions/actions_items.dmi' //placeholder
+	button_icon_state = "neckchop" //placeholder
+
+/datum/action/set_phase/Trigger() //Toggle the gloves phase between on or off.
+	if(owner.incapacitated())
+		to_chat(owner, "<span class='warning'>You can't use [name] while you're incapacitated!</span>")
+		return
+	if(isPhased == FALSE)
+		to_chat(owner, "<span class ='notice'>You activate the gauntlets, preparing them to phase!</span>")
+		isPhased = TRUE
+		owner.visible_message("[owner.name] activates their Blueshift Gauntlets with a low hum.")
```

Comment:
you could use the sound effect for turning a stasis bed on (NOT the sound effect for BUCKLING someone to a stasis bed) here for the humming noise

Reply:
Yeah! I was considering using that, there are a few other candidates but that one i like.